### PR TITLE
Added blocking tag

### PR DESCRIPTION
**Pull request by MrKomodoDragon**

### CODE REVIEW

**Comment by MrKomodoDragon** [comment]
So How is this?

File: bot/resources/tags/blocking.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# good
+async with aiohttp.ClientSession() as session:
+    async with session.get('http://aws.random.cat/meow') as r:
+        if r.status == 200:
+            js = await r.json()
+            await channel.send(js['file'])
+```
+
+**How can I find asynchronous modules?**
+Most python modules have an asynchronous implementation. For example, `sqlite3` has `aiosqlite`, and `praw` has `asycpraw`. You try searching on [PyPi](https://pypi.org). Or check out [this list](https://github.com/timofurrer/awesome-asyncio).
```
Comment (ChrisLovering) [inline bot/resources/tags/blocking.md:29]:
```suggestion
Most Python modules have an asynchronous implementation. For example, `sqlite3` has `aiosqlite`, and `praw` has `asycpraw`. You try searching on [PyPi](https://pypi.org), or check out [this curated list of Python asyncio frameworks](https://github.com/timofurrer/awesome-asyncio).
```
Grammar, and give the user info on what the link is to.

File: bot/resources/tags/blocking.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You can also ask here in this server for options.
```
Comment (ChrisLovering) [inline bot/resources/tags/blocking.md:30]:
```suggestion
You can also ask in <#630504881542791169> for options.
```
Link to the async topical channel

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

File: bot/resources/tags/blocking.md (hunk 1 above)
Comment (MrKomodoDragon) [reply bot/resources/tags/blocking.md:29]: Ok, good ideas!

File: bot/resources/tags/blocking.md (hunk 2 above)
Comment (MrKomodoDragon) [reply bot/resources/tags/blocking.md]: Done

**Comment by Xithrius** [comment]
@MrKomodoDragon when committing suggested changes, please change the commit message to describe what occurred within the update.

**Comment by MrKomodoDragon** [comment]
Ok, will do next time

File: bot/resources/tags/blocking.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+An asynchronous program doesn't wait for step to be finished before executing another one, but just keeps continuing executing other steps.
+
+It will also know what to do when the previous step finishes executing.
+
+Consider this example from [Miguel Grinberg’s 2017 PyCon talk](https://www.youtube.com/watch?t=4m29s&v=iG6fr81xHKA&feature=youtu.be), about playing multiple games of chess at once.
+
+
+**What does blocking mean?**
+
+In asynchronous programming, blocking calls are all the parts of your function that are not using `await`.
```
Comment (decorator-factory) [inline bot/resources/tags/blocking.md:12]: > In asynchronous programming, blocking calls are all the parts of your function that are not using `await`.
> 
> Not all forms of blocking are bad, and using blocking calls are inevitable, but make sure not to use too much.

I think this part would be a bit vague for someone who's new to async/await. `x + y` is a blocking call, but you wouldn't usually describe it as such. Maybe clarify that it applies mostly to blocking I/O (with examples such as `open` and `requests`), or to CPU-heavy tasks.

Comment (MrKomodoDragon) [reply bot/resources/tags/blocking.md:12]: OK

**Comment by Xithrius** [comment]
@MrKomodoDragon are you planning to finish this off?

**Comment by Xithrius** [comment]
Author hasn't responded to any pings. This PR can now be taken by any, with approval from a core developer.

**Comment by doublevcodes** [comment]
I'll take this

**Comment by doublevcodes** [comment]
How am I supposed to do this, would it entail creating a whole new pull request?

**Comment by Xithrius** [comment]
> How am I supposed to do this, would it entail creating a whole new pull request?

@doublevcodes that would make things much more simple. Be sure to link this PR in the same way you'd say that a PR closes an issue.

**Pull request closed without merging**